client/src/global.d.ts

```diff
@@ -1,6 +1,9 @@
 declare namespace google.accounts {
-    const id: {
-      initialize: (options: { client_id: string; callback: Function }) => void;
-      renderButton: (element: HTMLElement | null, options: { theme: string; size: string; text: string }) => void;
-    };
-  }
\ No newline at end of file
+  const id: {
+    initialize: (options: { client_id: string; callback: () => void }) => void;
+    renderButton: (
+      element: HTMLElement | null,
+      options: { theme: string; size: string; text: string }
+    ) => void;
+  };
```

Comment:
This doesn't look like it's being lint-tested against. You'll need to either create a new workflow in `.github/workflows` or find a way to add to the existing one. I would choose the former, as it's easier to manage and organize.